app/assets/javascripts/users.js

```diff
@@ -65,6 +65,11 @@ WebsiteOne.define('Users', function () {
         });
 
         $('#require_users_profile').modal({show: true});
+
+        // Enable bootstrap tooltips on non-touch devices only
+        if (!('ontouchstart' in document.documentElement)) {
+            $('[data-toggle="tooltip"]').tooltip({delay: { "show": 500, "hide": 100 }});
```

Comment:
Isn't that question a bit like asking "do we need to use bootstrap &lt;insert name of any bootstrap feature we currently use here&gt;"?   
Do we _need_ to use them?  No.  We _choose_ to use features of bootstrap because we like how they look or they make it quicker/easier to do something - so we use the grid layout, the nav bars and other things bootstrap provides.

I used bootstrap tooltips because they look a lot better (IMO) than the default ones but I can certainly remove them if you prefer the standard look - though I would say if you want WSO to showcase what we can do, then default tooltips is not going to wow anyone :)